device.mk

```diff
@@ -104,11 +104,21 @@ PRODUCT_COPY_FILES += \
 PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/ubuntu/ofono.override:system/halium/etc/init/ofono.override \
     $(LOCAL_PATH)/ubuntu/70-hammerhead.rules:system/halium/lib/udev/rules.d/70-android.rules \
+    $(LOCAL_PATH)/ubuntu/70-hammerhead.rules:system/halium/usr/lib/lxc-android-config/70-hammerhead.rules \
```

Comment:
You do not need to bindmount the same file thats already bound one line before- There will be no advantage to that.

Reply:
ystem/halium/lib/udev/rules.d/70-android.rules is replaced for system/halium/usr/lib/lxc-android-config/70-hammerhead.rules.
So i know that is replaced but i dosnt hurt.